### ISSUE TITLE
[miio] Add support for Mi Airpurifier Pro H (zhimi.airpurifier.vb2

### ISSUE THREAD

**Issue by theater** [openhab/openhab-addons#10174]
Hi,
As discussed in the community forums, here is the dump of humidifier data in order to add support for it in miio binding.
[test-zhimi.airpurifier.vb2-20210215-174145.txt](https://github.com/openhab/openhab-addons/files/5989319/test-zhimi.airpurifier.vb2-20210215-174145.txt)

Feel free to buzz me if you need me to test anything related to that before the actual PR.

Cheers,
Konstantin

**Comment by theater** [comment]
Hi Marcel,
log can be found here. Tried several channels. Power off/on works, the read only channels also seem to be working fine (tried the basic ones - temperature, humidity and ppm3.

I'm wondering if there is a way to query the API for the allowed values of Mode button? I guess it's some Enum or something like this and API should be able to return it but not sure if they implemented that
[miio.log](https://github.com/openhab/openhab-addons/files/5993934/miio.log)

Cheers,
Konstantin

**Comment by marcelrv** [comment]
Did you see the included encoding for mode options? Or Do you see in reality different values?
There is no (known) other automated way to query the modes other than I already did.
If this list is not fine, the only other alternative is to go, change the various modes in the app and see what is reported in OH as corresponding code

"channel": "mode",
				"type": "Number",
				"stateDescription": {
					"options": [
						{
							"value": "0",
							"label": "Auto"
						},
						{
							"value": "1",
							"label": "Sleep"
						},
						{
							"value": "2",
							"label": "Favorite"
						},
						{
							"value": "3",
							"label": "None"
						}
					]
				},

**Comment by marcelrv** [comment]
would you mind trying to link the other channels as well (you can do it in 1 go by using add to model button.
Than I know which ones to keep and which can go. (many times the online spec has channels that are actually not working)

**Comment by theater** [comment]
> Did you see the included encoding for mode options? Or Do you see in reality different values?
Actually I didn't see them, as I was looking in the logs and not the JSONs. Thanks !
In the UI they differ and semantics of 2 and 3 is different: 
0 is "Auto", 1 is "Night", 2 is "Manual", 3 is "Level3" (it's like max-fan speed)

**Comment by theater** [comment]
Hi,
here are fresh logs. When I restarted the bundle the thingss did not appear online due to missing handlers. Not sure if it's MIIO binding issue or is more general issue. Attached two logs. one is the bundle restart log and the other is fresh log from whole openhab restart after adding all points to the model as suggested. (hope I can remove them as easy as I added them later...)
Thanks for adding these so fast !
If anything else is needed, probably I'll have time for OH earliest at evening. :( 

Cheers,
K.
[bundle.restart.log](https://github.com/openhab/openhab-addons/files/5994293/bundle.restart.log)
[new_startup.log](https://github.com/openhab/openhab-addons/files/5994294/new_startup.log)
